source/extensions/quic_listeners/quiche/platform/quic_port_utils_impl.h

```diff
@@ -0,0 +1,10 @@
+
+#pragma once
+
+// NOLINT(namespace-envoy)
+
+// This file is part of the QUICHE platform implementation, and is not to be
+// consumed or referenced directly by other Envoy code. It serves purely as a
+// porting layer for QUICHE.
+
+#include "test/extensions/quic_listeners/quiche/platform/quic_port_utils_test_impl.h"
```

Comment:
Can you add a comment explaining this redirect for posterity? Thanks

Reply:
done